Library/ViewModels/SettingsAccountViewModelTests.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import XCTest
+import ReactiveSwift
+import Result
+import Prelude
+@testable import KsApi
+@testable import Library
+@testable import ReactiveExtensions_TestHelpers
+@testable import Kickstarter_Framework
+
+internal final class SettingsAccountViewModelTests: TestCase {
+  let vm = SettingsAccountViewModel()
+  let dismissCurrencyPicker = TestObserver<Void, NoError>()
+  let presentCurrencyPicker = TestObserver<Void, NoError>()
+  let reloadDataUser = TestObserver<User, NoError>()
+  let reloadDataCurrency = TestObserver<Currency, NoError>()
+  let showAlert = TestObserver<(), NoError>()
+  let updateCurrencyFailure = TestObserver<String, NoError>()
+
+  internal override func setUp() {
+    super.setUp()
+    self.vm.outputs.dismissCurrencyPicker.observe(self.dismissCurrencyPicker.observer)
+    self.vm.outputs.presentCurrencyPicker.observe(self.presentCurrencyPicker.observer)
+    self.vm.outputs.reloadData.map(first).observe(self.reloadDataUser.observer)
+    self.vm.outputs.reloadData.map(second).observe(self.reloadDataCurrency.observer)
+    self.vm.outputs.showAlert.observe(self.showAlert.observer)
+    self.vm.outputs.updateCurrencyFailure.observe(self.updateCurrencyFailure.observer)
+  }
+
+  func testReloadData() {
+    self.vm.inputs.viewDidLoad()
+    self.reloadDataUser.assertValueCount(1)
+    self.reloadDataCurrency.assertValueCount(1)
+    self.vm.inputs.showChangeCurrencyAlert(for: Currency.CHF)
+    self.vm.inputs.didConfirmChangeCurrency()
+    self.scheduler.advance()
```

Comment:
Can we assert here that `reloadDataUser` and `reloadDataCurrency` have a value count of 2?